tests/requirements/requirements_base.txt

```diff
@@ -23,6 +24,7 @@ unittest-xml-reporting==1.11.0
 
 # Unreleased django-cms 4.0 compatible packages
 https://github.com/django-cms/django-cms/tarball/release/4.0.1.x#egg=django-cms
-https://github.com/django-cms/djangocms-alias/tarball/master#egg=djangocms-alias
+https://github.com/joshyu/djangocms-alias/tarball/feat/django-42-compatible#egg=djangocms-alias
+https://github.com/FidelityInternational/djangocms-moderation/tarball/feature/django-42-compat#egg=djangocms-moderation
 https://github.com/django-cms/djangocms-text-ckeditor/tarball/support/4.0.x#egg=djangocms-text-ckeditor
```

Comment:
you should move `djangocms-text-ckeditor` to `django-3.2.txt` and `django-4.2.txt` and then use **version 5.15** in `django-4.2.txt`, so that you make sure all the test passed under django 4.2 environment we needed.

Reply:
Used Latest version in both and working fine now